tests/override_config_table/test_override_config_table.py

```diff
@@ -144,10 +166,17 @@ def load_minigraph_with_golden_full_config(duthost, full_config):
     for table in full_config:
         if table in NON_USER_CONFIG_TABLES:
             continue
-        pytest_assert(
-            full_config[table] == current_config[table],
-            "full config override fail! {}".format(table)
-        )
+
+        if table == "ACL_TABLE":
+            pytest_assert(
+                compare_dicts_ignore_list_order(full_config[table], current_config[table]),
+                "full config ACL_TABLE compare fail!"
+            )
+        else:
+            pytest_assert(
+                full_config[table] == current_config[table],
+                "full config override fail! {}".format(table)
```

Comment:
this test had never failed on backend. why was the ordering issue not seen in this case?

Reply:
[cluster('sonicrepodatadev.westus.kusto.windows.net').database('SonicTestData')](https://dataexplorer.azure.com/clusters/sonicrepodatadev.westus/databases/SonicTestData).TestReportUnionData
| where UploadTimestamp >= ago(250d)
// | where OSVersion contains "202205" 
| where TestbedName in ("vms18-t0-7050qx-acs-02", "vms18-t1-7050qx-acs-03") 
| where ModulePath contains "test_override_config_table"
| where Result !in ("skipped")
| extend Pipeline = strcat("https://dev.azure.com/mssonic/internal/_build/results?buildId=", BuildId)
| project TestbedName, StartTimeUTC, OSVersion, ModulePath,TestCase, Result, Summary, Pipeline
| order by StartTimeUTC desc

Hi @neethajohn , if you check the kusto. The t0 backend never succeed. t1 doesn't has this issue because it has not VLAN_MEMBER in t1 topo so it will not run the part of code to group ports in a set: https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-config-engine/minigraph.py#L1338-L1352